src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -1050,11 +1050,22 @@ def get_workload_identity_profile(self) -> Optional[ManagedClusterSecurityProfil
         # - False: sets by user, to disable the workload identity feature
         # - None: user unspecified, don't set the profile and let server side to backfill
         enable_workload_identity = self.raw_param.get("enable_workload_identity")
+        disable_workload_identity = self.raw_param.get("disable_workload_identity")
```

Comment:
how is this used in update code path?

Reply:
synced offline, as only one of these mutual exclusive options could be enabled,
> profile.enabled = bool(enable_workload_identity)

line 1074 does the conversion.

Reply:
`enable_workload_identity` and `disable_workload_identity` are exclusive and we have 

```
profile.enabled = bool(enable_workload_identity)
```

actually check whether workload identity is enabled or not.